txp/Pos/Txp/Logic/Local.hs

```diff
@@ -90,15 +91,18 @@ txProcessTransaction itw =
 
 -- | Unsafe version of 'txProcessTransaction' which doesn't take a
 -- lock. Can be used in tests.
-txProcessTransactionNoLock
-    :: ( TxpLocalWorkMode ctx m
-       , MempoolExt m ~ ()
-       )
-    => (TxId, TxAux) -> m (Either ToilVerFailure ())
+txProcessTransactionNoLock ::
+       forall ctx m. (TxpLocalWorkMode ctx m, MempoolExt m ~ ())
+    => (TxId, TxAux)
+    -> m (Either ToilVerFailure ())
 txProcessTransactionNoLock =
     txProcessTransactionAbstract
         buildProccessTxContext
-        processTx
+        processTxLocal
+  where
+    processTxLocal e t@(_,txAux) = do
+        either (throwError . ToilInconsistentTxAux . show) pure (runPVerify txAux)
```

Comment:
No, move it to `Toil.processTx` please. It's not used for VAR, only for local txp.

Reply:
Otherwise explorer's txp is kinda broken.

Reply:
Alright, thank you.